test/server/cards/02_SHD/upgrades/TopTarget.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe('Top Target', function() {
+    integration(function(contextRef) {
+        describe('Top Target\'s Bounty ability', function() {
+            it('should heal 4 damage from a base if the attached unit is not unique', function () {
```

Comment:
Suggestion: I think we may also want to test the scenario where the owner of the bounty has it placed on its own unit, and ensure the bounty is claimed by the opponent. Just to be sure bounty ownership does not affect anything.